Hibás tesztlefutás (GT4500Test)

Running hu.bme.mit.spaceship.GT4500Test
[ERROR] Tests run: 2, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.12 s <<< FAILURE! - in hu.bme.mit.spaceship.GT4500Test
[ERROR] fireTorpedo_All_Success  Time elapsed: 0.034 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: expected: <true> but was: <false>
	at hu.bme.mit.spaceship.GT4500Test.fireTorpedo_All_Success(GT4500Test.java:37)

Visszatérés true, az elvárt false lenne!

--- a/src/main/java/hu/bme/mit/spaceship/GT4500.java
+++ b/src/main/java/hu/bme/mit/spaceship/GT4500.java
@@ -79,6 +79,12 @@ public class GT4500 implements SpaceShip {
       case ALL:
         // try to fire both of the torpedo stores
         //TODO implement feature
+	if (! primaryTorpedoStore.isEmpty() && ! secondaryTorpedoStore.isEmpty())
+	{
+		firingSuccess = primaryTorpedoStore.fire(1);
+		firingSuccess &= secondaryTorpedoStore.fire(1);
+	}
+
 
         break;
     }

